ssl/ssl_lib.c

```diff
@@ -3600,6 +3600,13 @@ const SSL_CIPHER *SSL_get_current_cipher(const SSL *s)
     return (NULL);
 }
 
+const SSL_CIPHER *SSL_get_pending_cipher(const SSL *s)
+{
+    if (s->s3->tmp.new_cipher != NULL)
+        return s->s3->tmp.new_cipher;
+    return NULL;
```

Comment:
Couldn't this entire function just be:

```
return s->s3->tmp.new_cipher'
```

The explicit NULL checking isn't required when NULL is returned anyway.